_data/projects/FreeCourses.yml

```diff
@@ -0,0 +1,12 @@
+name: FreeCourses
+desc: A collection about Free Programming Courses
+site: https://freecourses.github.io/
+tags:
+- javascript
+- css
+- react
+- json
+- redux
+upforgrabs:
+  name: Add a Course
+  link: https://github.com/Leocardoso94/Free-Courses/issues/1
```

Comment:
This link should be to a list of curated issues, rather than an open-ended task. Without more guidance this doesn't feel like a good fit for up-for-grabs. @Leocardoso94 is this something you're able to refine?